addons/website/views/snippets/snippets.xml

```diff
@@ -481,9 +481,12 @@
     </div>
 
     <!-- Mobile display options -->
-    <div data-option-name="showMobile" data-selector="section .row > div"
+    <div data-js="showMobile" data-option-name="showMobile" data-selector="section .row > div, section"
          data-exclude=".s_col_no_resize.row > div">
-        <we-checkbox string="Shown On Mobile" data-select-class="d-none d-md-block|" data-no-preview="true"/>
+        <we-button class="fa fa-lg fa-mobile o_we_link o_we_mobile"
```

Comment:
I don't think this will look ok, aren't the other top option using fa-fw or something like that?